econet_electric_tank_water_heater.yaml

```diff
@@ -63,3 +56,7 @@ text_sensor:
     name: "Heating Element State"
     sensor_datapoint: HEATCTRL
     icon: "mdi:heating-coil"
+  - platform: econet
+    name: "Mode"
+    sensor_datapoint: WHTRMODE
+    entity_category: "diagnostic"
```

Comment:
Can this not be set via the Climate entity like with the other devices? If so, this sensor seems redundant. If not, I wonder if it should not be a diagnostic.

Reply:
From the discord discussion WHTRMODE was: Off-No Demand (0), Off-Delay Tmr(1), Vacation (11) and it seems to be read only. They were unsuccessful in modifying it, contrary to the WHTRCNFG that worked for them.

Reply:
Can we not get the vacation mode added in? Weird that it's non-contiguous...

Reply:
Added in the presets? Mark was unsuccessful setting it via either WHTRMODE or WHTRCNFG.

Reply:
This text sensor will just show the string value as returned. My guess is there are other modes between 1 and 11, like heating upper, heating lower, etc.

Reply:
Alright, well, this seems like an improvement over what's checked in currently.